tests/integration.py

```diff
@@ -0,0 +1,9 @@
+from bw_interface_schemas import load_graph
+
+
+def test_dump_graph(bike_as_graph):
+    assert bike_as_graph.model_dump()
+
+
+def test_construct_graph(bike_as_dict):
+    assert load_graph(bike_as_dict)
```

Comment:
suggestion (blocking): we could verify that every node in the fixture is present in the Graph object created by the load_graph function, as well as the edges, as a deeper test.